### PR TITLE
Merge servicing updates to main

### PR DESCRIPTION
**Pull request by twsouthwick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,12 @@ Select the subscription that will be used for the deployment:
 azd env set AZURE_SUBSCRIPTION_ID $AZURE_SUBSCRIPTION_ID
 ```
 
+(Optionally) Set your principal name:
```
Comment (adrianhall) [inline README.md:139]: When it's optional, we should describe why a person would want to set it - what does it get them?  When should they not set it (and what does that do?)

File: infra/core/compute/ubuntu-jumpbox.bicep [hunk 2]
```diff
@@ -204,29 +168,28 @@ resource virtualMachine 'Microsoft.Compute/virtualMachines@2021-11-01' = {
...
+    type: 'GuestAttestation'
+    typeHandlerVersion: '1.0'
     autoUpgradeMinorVersion: true
     enableAutomaticUpgrade: true
     settings: {
       AttestationConfig: {
         MaaSettings: {
-          maaEndpoint: maaEndpoint
-          maaTenantName: maaTenantName
+          maaEndpoint: substring('emptystring', 0, 0)
```
Comment (adrianhall) [inline infra/core/compute/ubuntu-jumpbox.bicep:191]: Isn't this just '' - can we remove the substring?

File: infra/core/security/front-door-with-waf.bicep [hunk 3]
```diff
@@ -36,6 +22,49 @@ type WAFRuleSet = {
   version: string
 }
 
+type CustomRule = {
```
Comment (adrianhall) [inline infra/core/security/front-door-with-waf.bicep:25]: Should we abstract this out into its own type file?

**Review by adrianhall — APPROVED** [review]

File: infra/core/compute/ubuntu-jumpbox.bicep (hunk 2 above)
Comment (twsouthwick) [reply infra/core/compute/ubuntu-jumpbox.bicep:191]: probably - not sure why it was like that. changing....

File: infra/core/compute/ubuntu-jumpbox.bicep [hunk 4]
```diff
@@ -46,19 +28,13 @@ param name string
 @description('The name of the linux PC.  By default, this will be automatically constructed by the resource name.')
```
Comment (coderandhiker) [inline infra/core/compute/ubuntu-jumpbox.bicep:28]: nit: consistency in naming - we're calling it a Virtual Machine elsewhere, not PC

**Review by coderandhiker — APPROVED** [review]

**Pull request merged**